packages/dd-trace/src/instrumenter.js

```diff
@@ -115,7 +115,19 @@ class Instrumenter {
       })
     })
 
-    shimmer.massWrap.call(this, nodules, names, wrapper)
+    shimmer.massWrap.call(this, nodules, names, function (original, name) {
```

Comment:
Can you add a comment referencing https://github.com/othiym23/shimmer/issues/19 so that we know why we did this?

Reply:
Sure, but it's not necessarily a bug since the library doesn't copy anything from the function, so it could be intended.